lib/tasks/avalon.rake

```diff
@@ -223,6 +223,16 @@ EOC
         end
       end
     end
+
+    desc "Starts Status Checking and Email Notification of Existing Batches"
+    task :ingest_status_check => :environment do
+      IngestFinished.perform_later
```

Comment:
If this is running every minute, I think we need to protect it from sending duplicate emails.  Maybe we could do this by wrapping it in a `WithLocking` like `avalon:batch:ingest` and changing it to `perform_now`?

Reply:
I just have it at one minute for testing, my intent was to back it off to every 15 minutes or so in production, no reason to hit our DB that hard, I'll back it off

Reply:
I think the main cost here is loading the rails context every minute.  It makes sense to be as responsive as possible when the batch is getting loaded but every 15 minutes on reporting completed is probably fine for now.  In that case, if we push it to 15 minutes then we could skip the locking and see if the issue ever comes up.